docs/quickstart/go.md

```diff
@@ -4,4 +4,184 @@ headline: Go Quick Start
 layout: docs
 sidenav: doc-side-quickstart-nav.html
 type: markdown
----
\ No newline at end of file
+---
+
+<p class="lead">This guide gets you started with gRPC in Go with a simple
+working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+#### Install gRPC
+To install gRPC on your system use the following command:
+```sh
+$ go get google.golang.org/grpc
+```
+
+#### Install Protocol Buffers v3
+
+While not mandatory, gRPC usually leverages Protocol Buffers v3 for service
+definitions and data serialization.
+
+1. Install the protoc compiler that is used to generate gRPC service code. The simplest way to do this is to download pre-compiled binaries for your platform here: https://github.com/google/protobuf/releases
+
+2. Next, install the protoc plugin for Go by following the instructions here: https://github.com/golang/protobuf#installation	
+  *  ``` $ go get -u github.com/golang/protobuf/{proto,protoc-gen-go} ```
+  *  The compiler plugin, protoc-gen-go, will be installed in $GOBIN, defaulting to $GOPATH/bin. It must be in your $PATH for the protocol compiler, protoc, to find it.
+     
+    ``` $ export PATH=$PATH:$GOPATH/bin ```
+
+##Download the example
+The grpc code that was fetched with ``` $ go get google.golang.org/grpc ``` also contains the examples. They can be found under the examples dir; $GOPATH/src/google.golang.org/grpc/examples
+
+## Build the example
+Change to the example directory
+
+```$ cd $GOPATH/src/google.golang.org/grpc/examples/helloworld```
+
+For gRPC services to work we want to compile the proto file to generate a pb.go file that contains the generated code which is then used by the server and client apps. 
```

Comment:
I realize that but I thought the users might want to know about how to actually generate that file. I can remove that if you find it extraneous.